go-runtime/schema/schema_integration_test.go

```diff
@@ -565,7 +567,8 @@ func testErrorReporting(t *testing.T) {
 		`146:6-45: enum discriminator "TypeEnum3" cannot contain exported methods`,
 		`149:6-35: enum discriminator "NoMethodsTypeEnum" must define at least one method`,
 		`161:3-14: unexpected token "d"`,
-		`168:2-62: can not publish directly to topics in other modules`,
+		// todo: do we want this? why do we support exporting topics if so?
+		//`168:2-62: can not publish directly to topics in other modules`,
```

Comment:
should this check be removed?

Reply:
Ah we still want this limitation. We export topics so that subscriptions in other modules can subscribe to them. But we don't want to let anyone publish to topics directly.

Reply:
ah ok cool, i'll restrict that via the generated external module so there's no `Publish` method at all

Reply:
actually that won't work. will keep the static check